### PR TITLE
Configuring with plone/meta

### PR DESCRIPTION
**Pull request by jensens**
test using PR https://github.com/plone/meta/pull/79

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: setup.cfg [hunk 1]
```diff
@@ -93,6 +73,10 @@ ignore =
     E203,
     # black takes care of spaces after commas
     E231,
-per-file-ignores =
-    src/plone/base/interfaces/__init__.py:F401
-    src/plone/base/permissions.py:F401,E402
```
Comment (gforcada) [inline setup.cfg:98]: Keep these lines so you don't have to add `  # noqa: F401` all over the place 😅

**Review by gforcada — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 1 above)
Comment (jensens) [reply setup.cfg:98]: This got overridden. `config-package.py` should not do that!

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by gforcada — APPROVED** [review]

**Pull request merged**